config_src/drivers/nuopc_cap/mom_cap.F90

```diff
@@ -452,7 +459,23 @@ subroutine InitializeAdvertise(gcomp, importState, exportState, clock, rc)
   CALL ESMF_TimeIntervalGet(TINT, S=DT_OCEAN, RC=rc)
   if (ChkErr(rc,__LINE__,u_FILE_u)) return
 
-  !TODO: next two lines not present in NCAR
+  !---------------------------------
+  ! openmp threads
+  !---------------------------------
+
+  call ESMF_VMGet(vm, pet=localPet, peCount=localPeCount, rc=rc)
+  if (ChkErr(rc,__LINE__,u_FILE_u)) return
+
+  if(localPeCount == 1) then
```

Comment:
The two instances of retrieving the attribute 'nthreads' are failing for EMC. They will need "isPresent,isSet" conditionals added. Thanks.

Reply:
Done. Thanks, @DeniseWorthen.